README.md

```diff
@@ -684,6 +684,203 @@ var callbacks = {
 BillForward.addPayVisionForm(formSelector, supportedCardBrands, wpwlOptions, callbacks);
 ```
 
+### Capture Bank Account
+
+For the moment the only bank account capture supported is in `stripe` using `ACH`.
+When capturing a bank account the flow can be divided in two steps:
+
+- Capturing
+- Verification
+
+Once the `Capture` has been made, `stripe` will deposit two random amounts on the bank account. These amounts have to be
```

Comment:
Regarding:

> `Capture`
> `Billforward`
> `payment method`
> state
> `Active`
> `html`

My preference is to use inline code formatting for:
- field names
- code
- text that benefits from monospaced formatting (i.e. dates)

One-by-one, my thoughts…

> `Capture`

I believe your intention was to style this as a "reference" (since it is one of the steps that you list).
Best thing I can think of is "emphasis": _Capture_ (in markdown, type: `_Capture_`)

> `Billforward`

We don't currently use any convention for styling our brand.
Probably it is sufficient to type it normally, but with studly case: BillForward

> `payment method`

Prefer PaymentMethod or `PaymentMethod`.
Also: be conscious that most payment gateways have their own concept of PaymentMethod.
I prefer to disambiguate (at the expense of brevity) by *always* qualifying:
- "The BillForward PaymentMethod is deleted"
- "The PaymentMethod in Stripe is deleted"

> state

This is a field on PaymentMethod, so should be: `state`.

> `Active`

This _could_ be okay. It is a constant on an enum type.
Consider also:
"Active" or `"Active"`,
Because at this end of the API, you are sending a _string_, not a constant.
Also consider how the sentence will read:
- …PaymentMethod `state` to `Active`
- …PaymentMethod `state` to `"Active"`
- …PaymentMethod `state` to "Active"
I think I prefer the final one ("Active"); it allows me to easily understand the difference between field name and field value.

> `html`

This is an initialism, so it should be upper-case: HTML.
If this is a reference (for example if you gave the user a choice of HTML or JS), you should also emphasize it:
_HTML_.